packages/@adobe/spectrum-css-temp/components/inputgroup/skin.css

```diff
@@ -217,3 +217,90 @@ governing permissions and limitations under the License.
     }
   }
 }
+@media (forced-colors: active) {
+  .spectrum-InputGroup {
+    --spectrum-dropdown-border-color-error: Highlight;
+    --spectrum-dropdown-border-color-key-focus: Highlight;
+    --spectrum-textfield-border-color-error: Highlight;
+    --spectrum-textfield-border-color-hover: Highlight;
+    --spectrum-textfield-border-color-key-focus: Highlight;
+    --spectrum-textfield-quiet-border-color: Canvas;
```

Comment:
Should this be Canvas or GreyText? Looks like quiet comboboxes are missing the bottom border right now.
![quietComboBox](https://user-images.githubusercontent.com/9661127/175699515-cf93d816-4a49-4d62-88fe-de94287734ec.PNG)